src/sql/workbench/contrib/notebook/browser/calloutDialog/imageCalloutDialog.ts

```diff
@@ -197,9 +197,17 @@ export class ImageCalloutDialog extends Modal {
 		this._register(styler.attachCheckboxStyler(this._imageEmbedCheckbox, this._themeService));
 	}
 
+	private readonly doubleQuotesRegex = /^\"(.*)\"$/;
 	public insert(): void {
 		this.hide('ok');
 		let imgPath = this._imageUrlInputBox.value;
+
+		// Remove any quotes around URL
+		let matches = this.doubleQuotesRegex.exec(imgPath);
+		if (matches && matches[1]) {
```

Comment:
`if (matches?.[1])`

Reply:
Technically works, but it looks a little weird when using it with an array index.